dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.cpp

```diff
@@ -982,9 +980,6 @@ void FindDDLMViaEnumeration(
         version.Revision = packageVersion.Revision;
         if (version.Version < minVersion.Version)
         {
-            (void)LOG_HR_MSG(MDD_E_BOOTSTRAP_INITIALIZE_DDLM_SCAN_NO_MATCH,
-                             "Bootstrap.Intitialize: %ls not applicable. Version doesn't match MinVersion criteria (%ls)",
-                             packageFullName.c_str(), criteria.get());
             continue;
```

Comment:
why did we remove logging altogether here (and below)?

Reply:
I can add the comments back. Those messages were diagnostic logging and would add 5-10 lines on each build if you have multiple Winappsdk packages -  main, ddlm, etc. It is just a search result and I didn't want to increase noise in user visible area. Hence, decided to remove unnecessary logging. We are not losing any helpful debugging info from this. This code tells which packages it considered to be invalid. That list can be long.

Reply:
Thanks for explanation - I'm fine with the change, just wanted to clarify.